tests/src/Interop/StringMarshalling/LPTSTR/LPTSTRTestNative.cpp

```diff
@@ -33,6 +33,20 @@ extern "C" LPWSTR ReturnErrString()
 }
 
 //Test Method1
+extern "C" DLL_EXPORT LPWSTR Marshal_In(/*[In]*/LPWSTR s)
+{
+    //Check the Input
+    size_t len = wcslen(s);
```

Comment:
```suggestion
    size_t len = TP_slen(s);
```

Reply:
I have found what was making the musl CI legs to fail. You have recently modified the tests to use `TP_xxx` functions instead of wide string ones redefined to use char16_t. But since this PR predates this change, it was still using the redefined wide string functions. Alpine seems to be the only Linux where it causes the build to fail for some reason.

Reply:
I don't know how I missed that... Thanks for your help!